libs/product/src/index.ts

```diff
@@ -8,8 +8,8 @@ export { DaffProductImage } from './models/product-image';
 export { DaffBundledProduct } from './models/bundled-product';
 export { DaffBundledProductItem, DaffPriceTypeEnum, DaffBundledProductItemOption } from './models/bundled-product-item';
 
-import * as fromProduct from './reducers/index';
```

Comment:
Is this temporarily necessary (but deprecated) for backwards compatibility?

Reply:
no, I made the fixes to demo and category